modules/claims_api/app/workers/claims_api/vmbs_updater.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'bgs'
+
+module ClaimsApi
+  class # frozen_string_literal: true
+
+    require 'sidekiq'
+    require 'bgs'
+    
+    module ClaimsApi
+      class VBMSUpdater
+        include Sidekiq::Worker
+    
+        def perform(power_of_attorney_id)
+          poa_form = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
+          service = BGS::Services.new(
+            external_uid: poa_form.external_uid,
+            external_key: poa_form.external_key
+          )
+          service.corporate_update.update_poa_access(
+            participant_id: target_veteran.participant_id,
+            poa_code: form_data.form_data.dig('serviceOrganization', 'poaCode'),
```

Comment:
is this supposed to be `poa_form.form_data`?

Reply:
ahh yeah, good catch